applications/callflow/src/module/cf_call_forward.erl

```diff
@@ -87,8 +87,8 @@ cf_menu(#callfwd{keys=#keys{menu_toggle_cf=Toggle
                 }=CF, CaptureGroup, Call) ->
     lager:info("playing call forwarding menu"),
     Prompt = case Enabled of
-                 'true' -> kz_media_util:get_prompt(<<"cf-enabled_menu">>, Call);
-                 'false' -> kz_media_util:get_prompt(<<"cf-disabled_menu">>, Call)
+                 'true' -> kz_media_util:get_prompt(<<"cf-enabled_menu">>, kapps_call:account_id(Call));
+                 'false' -> kz_media_util:get_prompt(<<"cf-disabled_menu">>, kapps_call:account_id(Call))
```

Comment:
@jamesaimonetti i think this should pass the `Language` of the call and not the accountid